mlrun/datastore/base.py

```diff
@@ -193,14 +193,15 @@ def reader(*args, **kwargs):
 
         fs = self.get_filesystem()
         if fs:
```

Comment:
@hayesgb i see that fs (`self.get_filesystem()`) = `fsspec.filesystem("az", **self.get_storage_options())` , so why do we need to pass the storage_options again to dask?

Reply:
When passing a single parquet file (rather than a directory) but loading with Dask, the below by-passes the loading of storage_options.

Reply:
but the single file option uses the fsspec file handle (`fs.open()`) which is supposed to incorporate the options initialized by the `fsspec.filesystem()` i thought?

Reply:
It does, but it returns a Pandas DataFrame.  If the code within the consuming Python function expects a Dask DataFrame, then it causes problems.

Reply:
@yaronha -- Are you comfortable with this?

Reply:
Per discusion, explicitly import dask.dataframe as dd, then load to dask if df_module is dd on line 198.